docroot/modules/custom/nvli_custom/src/ResourceCount.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\nvli_custom;
+
+/**
+ * Class ResourceCount.
+ *
+ * @package Drupal\nvli_custom
+ */
+class ResourceCount {
```

Comment:
@ashishdalvi Can we choose a more generic name for this service. Something like ResourceManager?